CourseGrab/Home/HomeTableViewCell.swift

```diff
@@ -92,7 +92,7 @@ class HomeTableViewCell: UITableViewCell {
             make.top.equalTo(sectionLabel.snp.bottom).offset(12)
             make.leading.equalTo(16)
```

Comment:
should the `leading` here be relative to superview?

Reply:
@danielVebman

Reply:
@kevinchan159 I think `make.leading.equalToSuperview().offset(16)` is equivalent to `make.leading.equalTo(16)`, except the latter is more concise.

Reply:
oh wow thats neat i didn't know that